MapboxNavigation/MGLMapView.swift

```diff
@@ -7,33 +7,86 @@ import MapboxCoreNavigation
+     - parameter layerIdentifier: Identifier of the layer in the tile set; in other words, a source layer identifier. Not to be confused with a style layer.
+     */
+    func showsTileSet(withIdentifier tileSetIdentifier: String, layerIdentifier: String) -> Bool {
+        guard let style = style else {
+            return false
+        }
+        
+        let incidentsSourceIdentifiers = sourceIdentifiers(forTileSetIdentifier: tileSetIdentifier)
+        for layer in style.layers {
+            if let layer = layer as? MGLVectorStyleLayer, let sourceIdentifier = layer.sourceIdentifier {
+                if incidentsSourceIdentifiers.contains(sourceIdentifier) && layer.sourceLayerIdentifier == layerIdentifier {
+                    return layer.isVisible
                 }
             }
-            return false
         }
-        set {
-            if let style = style {
-                for layer in style.layers {
-                    if let layer = layer as? MGLForegroundStyleLayer {
-                        if layer.sourceIdentifier == "mapbox://mapbox.mapbox-traffic-v1" {
-                            layer.isVisible = newValue
-                        }
-                    }
+        return false
+    }
+    
+    /**
+     Shows or hides data from the given tile set layer.
+     
+     - parameter tileSetIdentifier: Identifier of the tile set in the form `user.tileset`.
+     - parameter layerIdentifier: Identifier of the layer in the tile set; in other words, a source layer identifier. Not to be confused with a style layer.
+     */
+    func setShowsTileSet(_ isVisible: Bool, withIdentifier tileSetIdentifier: String, layerIdentifier: String) {
+        guard let style = style else {
+            return
+        }
+        
+        let incidentsSourceIdentifiers = sourceIdentifiers(forTileSetIdentifier: tileSetIdentifier)
+        for layer in style.layers {
+            if let layer = layer as? MGLVectorStyleLayer, let sourceIdentifier = layer.sourceIdentifier {
+                if incidentsSourceIdentifiers.contains(sourceIdentifier) && layer.sourceLayerIdentifier == layerIdentifier {
+                    layer.isVisible = isVisible
                 }
             }
         }
     }
+
+    /**
+     A Boolean value indicating whether traffic congestion lines are visible in the map view’s style.
+     */
+    public var showsTraffic: Bool {
+        get {
+            return showsTileSet(withIdentifier: "mapbox.mapbox-traffic-v1", layerIdentifier: "traffic")
+        }
+        set {
+            setShowsTileSet(newValue, withIdentifier: "mapbox.mapbox-traffic-v1", layerIdentifier: "traffic")
+        }
+    }
+    
+    /**
+     A Boolean value indicating whether incidents, such as road closures and detours, are visible in the map view’s style.
+     */
+    public var showsIncidents: Bool {
+        get {
+            return showsTileSet(withIdentifier: "mapbox.mapbox-incidents-v1", layerIdentifier: "incidents")
```

Comment:
I jumped the gun and merged this PR without updating the source layer identifier to match a last-minute change to the tileset. Fixed in c7ca0425ede884d88fe51d30d672ab84df3bd063.